src/commands/sync.ts

```diff
@@ -197,25 +202,24 @@ export default class Sync extends BaseCommand {
     this.debug("starting");
     await fs.ensureDir(this.dir);
 
-    try {
-      this.metadata = await fs.readJson(this.absolute(".ggt", "sync.json"));
-    } catch (error) {
-      // use defaults if the metadata file doesn't exist
-      ignoreEnoent(error);
-
-      const d = await fs.opendir(this.dir);
-      if ((await d.read()) != null) {
-        this.warn("Could not find .ggt/sync.json in a non empty directory");
+    if (await isEmptyDir(this.dir)) {
+      // use default metadata if the directory is empty
```

Comment:
Instead of having an empty branch with a comment I think it might be more clear to just show the case where the dir is not empty.

```javascript
if (!(await isEmptyDir(this.dir))) {
  try {
    // ...
  }
}
```